assets/js/app.js

```diff
@@ -0,0 +1,124 @@
+const player = (name, marker) => ({ name, marker });
+
+const gameboard = () => ({
+  arrayboard: ['', '', '',
+    '', '', '',
+    '', '', ''],
+});
+
+class TicTacToeGame {
```

Comment:
I've realized that you have DOM manipulation and aPplication logic tasks in the same class as an file.
Please remember that you ought to separate the two.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from pathwright.

Reply:
thanks i will split and separate the dom and the logic